pkg/apis/policy/v1alpha1/propagation_types.go

```diff
@@ -173,6 +173,13 @@ type PropagationSpec struct {
 	// +kubebuilder:validation:Enum=Lazy
 	// +optional
 	ActivationPreference ActivationPreference `json:"activationPreference,omitempty"`
+
+	// Suspend tells the controller to suspend subsequent executions.
```

Comment:
Whether the suspension operation is transparently transferred to the `work` object or the policy execution is suspended.

Reply:
The suspension operation is transparently transferred to the `work` object.

Is this better?
```suggestion
	// Suspend will pause all work objects to pause propagation to member clusters.
```

Reply:
I wonder if what really needs to be paused is the propagation of `work`. Pauses on pp and rb are to facilitate the transfer of user intent to `work`.

Reply:
Please ignore my question above. I see that's how you've implemented it now.